assets/js/base/context/tsconfig.json

```
@@ -10,7 +10,8 @@
 		"../../base/utils/",
 		"../../data/",
 		"../../types/",
-		"../components"
+		"../components",
+		"../../blocks/shared/payment-methods"
```

Do you mean this instead?
```suggestion
		"../../blocks/cart-checkout-shared/payment-methods"
```